app/src/main/java/com/example/pinterbest/adapters/PinFeedHomeAdapter.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.pinterbest.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.pinterbest.R
+import com.example.pinterbest.data.PinObject
+
+class PinFeedHomeAdapter(private val pinObjects: List<PinObject>) :
+    RecyclerView.Adapter<PinFeedHomeAdapter.ViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val view = LayoutInflater.from(parent.context)
+            .inflate(R.layout.view_holder_home_feed, parent, false)
+        return ViewHolder(view)
+    }
```

Comment:
If you use view binding in Fragments, let's use it in Adapters & ViewHolders too